### ISSUE TITLE
AttributeError: 'LogRecord' object has no attribute 'asctime'

### ISSUE THREAD

**Issue by artplan1** [ScrapeOps/scrapeops-scrapy-sdk#12]
```python
Traceback (most recent call last):
File \"/usr/local/lib/python3.11/site-packages/scrapeops_scrapy/core/error_logger.py\", line 141, in emit
dateTime = record.asctime
^^^^^^^^^^^^^^
AttributeError: 'LogRecord' object has no attribute 'asctime'
```

**Comment by bemayr** [comment]
I just stumbled upon this same problem after an unfortunate chain of dependency issues.
After trying to deploy a Scrapy project with the ScrapeOps integration to Zyte, the deploy failed, because of incompatible versions of `urllib3`.
```
botocore 1.31.49 has requirement urllib3<1.27,>=1.25.4, but you have urllib3 2.1.0
```
After digging into this I suspect that Zyte internally uses botocore for their S3 integrations and this issue is related to https://github.com/boto/botocore/issues/2926 (which is a pity).
So I started to downgrade my elastic integration, because it caused issues with `urllib3` as well, but found out that this package conflicts with `urllib3` as well.
As introduced in https://github.com/ScrapeOps/scrapeops-scrapy-sdk/commit/6e0e2113c5ff9b8c7a970bdfa11942dc8ad15c32, version **0.5.4** now relies on 2.1.0 of `urllib3` which raises the conflict.

So, I suspect that @artplan1 uses version **0.5.3** of this package, which raises this error because of the missing `if hasattr(record, 'message'):` in _error_logger.py_.
Version 0.5.4 works without a problem locally. 👏
